core/prop/prop.go

```diff
@@ -0,0 +1,18 @@
+package prop
```

Comment:
You'll want to turn go autoformat on: Add let g:go_fmt_autosave = 1 to your .vimrc after you install https://github.com/fatih/vim-go